src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java

```diff
@@ -16,6 +16,8 @@
   public static final String ARTICLE_NAME_PREFIX = "QAarticle";
   public static final String ARTICLE_TEXT =
       "Lorem ipsum dolor sit amet, consectetur adipiscing elit.";
+  public static final String ARTICLE_TEXT_DROZDY =
```

Comment:
Is there any reason for that peculiar name? (I'm not saying it isn't nice 😉  🐦 )